setup.py

```diff
@@ -15,8 +15,6 @@
 import os
 from setuptools import setup
 
-# Hack required to allow compat to not fail when pypiwin32 isn't found
-os.environ["PYINSTALLER_NO_PYWIN32_FAILURE"] = "1"
```

Comment:
From what I understand it's a hard dependency on Windows https://github.com/pyinstaller/pyinstaller/blob/617de3bb46280c77b391e0c29ff0c07f316a600d/setup.py#L33-L34 
so that would make sense. 

pip should pull it as a dependency though, since it's passed as `install_requires` to `setup()`?

Reply:
Aww right, because `setup.py` imports `PyInstaller.compat` which will fail if `pywin32-ctypes` is not installed. That was not intended. Will make another PR with a fix.

Reply:
Actually reverting this PR appears to be the way to go, since the only changes it contain are related to this hack.  Please feel free to do so.